catalog-info.yaml

```diff
@@ -6,7 +6,13 @@ kind: Component
 metadata:
   name: 'frontend-components-tinymce-advanced-plugins'
   description: "A set of tinymce plugins to use in react applications."
+  links:
+    - url: "https://github.com/openedx/frontend-components-tinymce-advanced-plugins/blob/master/README.rst"
+      title: "Documentation"
+      icon: "Article"
+  annotations:
+    openedx.org/arch-interest-groups: "angonz"
+    openedx.org/release: "master"
```

Comment:
```suggestion
```
You don't need to tag this repo at release, it's a library that gets pulled into one of the MFEs which is tagged.